### PR TITLE
Caeser1996 roc 5

### PR DESCRIPTION
**Pull request by caeser1996**
## Describe your changes
- Worked on creating the repository Surreadb repository
- Worked on writing test cases
## Issue ticket number and link
ROC-5 - https://freelancedevelopercoach.atlassian.net/browse/ROC-5
## Checklist before requesting a review
- [X] I have performed a self-review of my code

### CODE REVIEW

File: rococo/repositories/surreal_db_repository.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import json
+from typing import Type
+
+from rococo.data.surrealdb import SurrealDbAdapter
```
Comment (afeef) [inline rococo/repositories/surreal_db_repository.py:4]: @caeser1996 Can we shorten the imports to just import from package?

i. e `roco.data import SurrealDbAdapter`

File: tests/surreal_db_repository_test.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import json
+import unittest
+from unittest.mock import MagicMock, patch
+
+from rococo.models.versioned_model import VersionedModel
+from rococo.repositories.surreal_db_repository import SurrealDbRepository
```
Comment (afeef) [inline tests/surreal_db_repository_test.py:6]: @caeser1996 Same as above

**Review by afeef — CHANGES_REQUESTED** [review]

File: rococo/repositories/surreal_db_repository.py (hunk 1 above)
Comment (caeser1996) [reply rococo/repositories/surreal_db_repository.py:4]: from rococo.data import SurrealDbAdapter
from rococo.messaging import MessageAdapter
from rococo.models import VersionedModel
from rococo.repositories import BaseRepository

Like this ?

Comment (afeef) [reply rococo/repositories/surreal_db_repository.py]: Yes

Comment (caeser1996) [reply rococo/repositories/surreal_db_repository.py]: Done sir

File: tests/surreal_db_repository_test.py (hunk 2 above)
Comment (afeef) [reply tests/surreal_db_repository_test.py:6]: @caeser1996 I think you missed this line 😄

File: rococo/repositories/surreal_db_repository.py (hunk 1 above)
Comment (sinisarudan) [reply rococo/repositories/surreal_db_repository.py]: Yes, I've put to my Todo list too, bc I've seen some of those long ones still hanging around ;)

File: tests/surreal_db_repository_test.py (hunk 2 above)
Comment (caeser1996) [reply tests/surreal_db_repository_test.py]: Please check now

**Review by afeef — APPROVED** [review]

**Comment by sinisarudan** [comment]
I've resolved the setup.py conflict to `version='0.1.20'` that I've seen @caeser1996 you've bumped to

**Pull request merged**